app/components/concept-scheme-uri-selector.js

```diff
@@ -56,7 +56,11 @@ export default class ConceptSchemeUriSelectorComponent extends Component {
       },
     });
 
-    this.options = sortObjectsOnProperty([...conceptSchemes], 'label');
+    const notArchivedConceptSchemes = [...conceptSchemes].filter(
+      (scheme) => !scheme.isArchived
+    );
+
+    this.options = sortObjectsOnProperty(notArchivedConceptSchemes, 'label');
```

Comment:
@ZijneDwaasheid just thought about this that the archived conceptschemes will still be shown in the concept scheme selector even if they are archived.. 

Added this filter method to only get private and public conceptschemes that are not archived 👌🏻  when a form is created with an archived conceptscheme it will still show in the form preview and test as it is still in the database :)

Sorry about this.. Could you check it again :)

Reply:
Maybe test not yet as it is not using the new user-test page yet 🙈